packages/components/src/text/test/text.styles.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * External dependencies
+ */
+import { mount } from 'enzyme';
+/**
+ * Internal dependencies
+ */
+import { Text } from '../text.styles';
+
+const variants = [ 'title.large', 'title.medium', 'title.small', 'subtitle', 'subtitle.small', 'body', 'body.small', 'button', 'caption', 'label' ];
+
+describe( 'Text', () => {
+	variants.forEach( ( variant ) => {
+		// FIXME: replace with visual regression test
```

Comment:
Is there an issue opened with the proposal how to integrate visual regression tests?

Reply:
How about a PR? 😁
https://github.com/WordPress/gutenberg/pull/18797

I don't think we require these tests if we have Visual Regression Testing - nobody really cares about the markup/css used but how it displays.

Reply:
Nice, you can now remove those code comments 😃

Reply:
I've removed the two test files since there's a story displaying each of the variants and they'll be covered by both snapshot tests and 🤞visual regression tests.